diffrax/__init__.py

```diff
@@ -4,6 +4,7 @@
     AbstractAdjoint as AbstractAdjoint,
     BacksolveAdjoint as BacksolveAdjoint,
     DirectAdjoint as DirectAdjoint,
+    ForwardAdjoint as ForwardAdjoint,
```

Comment:
Nit: we really shouldn't call this 'adjoint' as that word refers specifically to reverse-mode autodifferentiation. Maybe `ForwardMode` instead?

Reply:
Haha see I should really dig deeper into the theory of this! 

Should we then rename the section in the documentation? Maybe "Differentiating through a solve"? `ForwardMode` also inherits from `AbstractAdjoint`. 

The easiest thing to do would be to simply add a comment to the documentation of `ForwardMode`, that it is not really an adjoint.